### PR TITLE
Add mail_layout_preview

### PR DESCRIPTION
**Pull request by simahawk**
Replaces #472

### CODE REVIEW

File: mail_layout_preview/wizard/email_template_preview.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```
Comment (i-vyshnevska) [inline mail_layout_preview/wizard/email_template_preview.py]:
```suggestion
# Copyright 2020 Simone Orsi - Camptocamp SA
# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
```

**Review by i-vyshnevska — APPROVED** [review]
Functional and code review +

**Review by hbrunn — CHANGES_REQUESTED** [review]
nice code, but I miss tests

**Comment by simahawk** [comment]
@hbrunn took me a while to get back to this :sweat_smile: 
Tests: there you go! :wink:

**Pull request merged**